packages/page-claims/src/Warning.tsx

```diff
@@ -31,10 +31,18 @@ function Warning ({ className }: Props): React.ReactElement<Props> | null {
       <div className={className}>
         {
           t(
-            'You need to sign an attestation for the following account(s): {{needsAttestArray}}',
-            { replace: { needsAttestArray } }
+            'You need to sign an attestation for the following {{account}}:',
+            { replace: { account: needsAttestArray.length > 1 ? 'accounts' : 'account' } }
```

Comment:
not 100% sure about how clever this is for translation. Can I use a `t('')` in a translation's `replace`?

Reply:
Not a good idea - you assume all languages has an english-like construct. The sanest approach (since it is a whole sentence), would be to have 2 full strings.

Reply:
Got it thanks, I made the changes.